llama-index-integrations/llms/llama-index-llms-bedrock-converse/llama_index/llms/bedrock_converse/base.py

```diff
@@ -385,32 +385,30 @@ def stream_complete(
     @llm_chat_callback()
     async def achat(
         self, messages: Sequence[ChatMessage], **kwargs: Any
-    ) -> NotImplementedError:
-        raise NotImplementedError("Async chat is not supported for Bedrock Converse.")
+    ) -> ChatResponse:
+        # TODO convert to async; do synchronous chat for now
+        return self.chat(messages, **kwargs)
```

Comment:
ngl, fake async kind of grinds mine (and a lot of peoples) gears. Time is much better spent just implmenting actual async

I'm pretty sure its as simple as using an async boto3 session, I just don't have access to bedrock to test